care/facility/api/viewsets/bed.py

```diff
@@ -240,3 +241,44 @@ def get_queryset(self):
             allowed_facilities = get_accessible_facilities(user)
             queryset = queryset.filter(bed__facility__id__in=allowed_facilities)
         return queryset
+
+    @extend_schema(
+        description="Toggle patient privacy",
+        responses={status.HTTP_200_OK: None},
```

Comment:
The API would be better usable if we respond with the new privacy state instead of None